src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -292,6 +302,47 @@ private bool IsManifestIdOutdated(string workloadManifestDir)
             return _outdatedManifestIds.Contains(manifestId);
         }
 
+        private Dictionary<string, WorkloadSet> GetAvailableWorkloadSets()
```

Comment:
Does 'available' mean that the workload set is on the disk and of the same feature band of the running SDK? It could be beneficial to add a docstring for this.